boards/DCM/Src/App/states/App_InitState.c

```diff
@@ -60,6 +60,12 @@ static void InitStateRunOnTick100Hz(struct StateMachine *const state_machine)
     return;
 #endif
 
+#ifdef CONFIG_EEPROM
+    App_InverterSwitches_TurnOnRight(inverter_switches);
+    App_InverterSwitches_TurnOnLeft(inverter_switches);
+    return;
+#endif
```

Comment:
This would mean you have to recompile the code without the CONFIG_EEPROM flag when you want to run after configuring the EEPROM which may be a bit inconvenient

Reply:
Think there should be a more convenient way to do this

Reply:
I could add another CAN message that'll turn on/off the inverter switches manually. What are your thoughts?

Reply:
Yeah i think using CAN would be the way. Will probably need CAN msgs to disable the inv faults too after config?

Reply:
I've added two CAN msgs which the user through PCAN can simply use to clear the inverter faults and turn on/off the inverter switches. I think this is the most straightforward approach as we avoid having to code an additional state and unnecessary logic. Let me know what you think